src/Careerbuilder.php

```diff
@@ -180,6 +183,52 @@ function ($value, $key) use (&$queryString) {
     }
 
     /**
+     * Get min and max salary numbers from string
+     *
+     * @return array
+     */
+    public function parseSalariesFromString($input = null)
```

Comment:
This guy is U G L Y and it ain't got no alibi! Scrut is giving it a score of D. Is there some refactoring that can be done here?